tests/functional/test_project.py

```diff
@@ -627,6 +627,22 @@ def test_from_python_library(self):
         assert "site-packages" in str(web3.path)
         assert web3.path.name == "web3"
 
+    def test_hash(self, with_dependencies_project_path, project_from_manifest):
+        """
+        Show we can use projects in sets.
+        """
+        project_0 = Project(with_dependencies_project_path)
+        project_1 = Project.from_python_library("web3")
```

Comment:
does web3 provide manifests?

Reply:
no, but you can create a "project" from any python package this way, even if empty.